mln/migrations/0024_networkermessagetrigger.py

```diff
@@ -13,16 +13,29 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='NetworkerTrigger',
+            fields=[
+                ('response', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, related_name='trigger', serialize=False, to='mln.messagebody')),
+                ('networker', models.ForeignKey(limit_choices_to={'profile__is_networker': True}, on_delete=django.db.models.deletion.CASCADE, related_name='+', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.RenameModel(
+            old_name='NetworkerMessageAttachmentDev',
+            new_name='NetworkerMessageAttachment',
+        ),
+        migrations.AlterField(
+            model_name='networkermessageattachment',
+            name='trigger',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='attachments', to='mln.networkertrigger'),
+        ),
```

Comment:
You can't do this, this would result in django being unable to associate the attachments with triggers since the "dev triggers" the attachments pointed to would no longer be referenced, and our data would be lost. Either make separate `NetworkerMessageAttachment` and `NetworkerMessageAttachmentDev` models or merge `NetworkerMessageTrigger` and `NetworkerMessageTriggerDev` into one model.